scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -414,8 +414,8 @@ export function getAutoTaggingComponent(superdesk: ISuperdesk, label: string) {
 
                                 let allGrouped = OrderedMap<string, JSX.Element>();
 
-                                othersGrouped.map((tags, groupId) =>
-                                    allGrouped = allGrouped.set(groupId,
+                                othersGrouped.forEach((tags, groupId) =>
+                                    tags && groupId ? allGrouped = allGrouped.set(groupId,
```

Comment:
That ternary is a bit confusing, it'd be more readable to use a simple if
```
othersGrouped.forEach((tags, groupId) => {
if (tags != null && groupId != null) {
    allGrouped = allGrouped.set(groupId, ...);
}
});
```

Can `tags` or `groupId` be `null` though? That map shouldn't have empty items

Reply:
Nope, but is showing error it is possible null, so just to avoid warning.